cwl/src/main/scala/cwl/CwlExpressionCommandPart.scala

```diff
@@ -5,14 +5,15 @@ import wom.CommandPart
 import wom.expression.IoFunctionSet
 import CwlWomExpression._
 import wdl4s.cwl.EvaluateExpression
+import wom.graph.LocalName
 
 
 case class CwlExpressionCommandPart(expr: Expression) extends CommandPart {
-  override def instantiate(inputsMap: Map[String, WdlValue],
-                            functions: IoFunctionSet,
-                            valueMapper: (WdlValue) => WdlValue): String = {
+  override def instantiate(inputsMap: Map[LocalName, WdlValue],
+                           functions: IoFunctionSet,
+                           valueMapper: (WdlValue) => WdlValue): String = {
 
-    val pc = ParameterContext.Empty.withInputs(inputsMap, functions)
+    val pc = ParameterContext.Empty.withInputs(inputsMap.map({ case (key, value) => key.asString -> value }), functions)
```

Comment:
Well I was trying to make it harder to get the String value so that we'd try to work with `LocalName` and `FullyQualifiedName` to get more type safety but yeah it seems a bit counter productive :p

Reply:
this line of commenting reminded me i keep meaning to say that i'd love the WOM effort to be the impetus for properly strongly typing LQNs and FQNs everywhere ...

Reply:
Yeah that's an effort towards that